src/angular/pagination/navigation/navigation.scss

```diff
@@ -4,20 +4,20 @@ $navigationIconTextMargin: 4;
 
 .sbb-navigation {
   display: flex;
+  margin: 0;
```

Comment:
Is there a need to manually set the margin to 0? Do we inherit it from somewhere?

Reply:
Yes from ul / li styles. I will add a comment do clarify